apps/styleguide/src/layout/MolgenisMenu.vue

```diff
@@ -76,6 +76,11 @@ export default {
       return this.items.filter(this.permitted);
     },
     homeUrl() {
+      // Editor, Manager and Admin back to cental, other users to the first permitted menu item
+      if (this.isUserEditor() || this.isUserManager() || this.isUserAdmin()) {
```

Comment:
I don't think this is the desired behavior. I think that behavior should be consistent between users otherwise an 'manager' gets complaints from a 'viewer' and cannot reproduce. I am of opinion that when we rollout the bread crumb then logo should always go back first menu item. If you want to go elsewhere you can use breadcrumb. We could even consider to disable action on logo completely.